projects/gen-ai.md

```diff
@@ -0,0 +1,143 @@
+
+By providing GenAI community with the space under OpenTelemetry we will be able to start hosting vendor-agnostic instrumentation libraries (such as WIP one [OpenAI Python](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2759)) and eventually we will be able to move popular OpenLLMetry-owned `opentelemetry-instrumentation-*` libraries to OTel (or to upstream libraries).
+
+It helps GenAI observability community to
+
+- find solutions for new observability problems such as verbose events or correlation with async response evaluations
+- discuss implementation challenges
+- coordinate and prioritize common efforts
+
+### Goals, objectives, and requirements
+
+GenAI observability project is a new OTel cross-cutting SIG that intends to provide the GenAI community with a space to evolve and maintain GenAI related instrumentations in a **vendor-agnostic way**.
+
+It includes:
+
+- LLM Semantic Conventions
+- Instrumentations for GenAI client libraries, frameworks, or other components.
+
+The eventual goal is to contribute instrumentation libraries to upstream components or to otel-contrib repos when upstream contributions are not possible.
+
+During initial development and experimentation, we may need to release libraries more frequently than it's possible for contrib and then may temporarily host instrumentation libraries in a `opentelemetry-{language}-genai` repository(ies).
+
+This would help GenAI observability vendors that will depend on these libraries to patch issues and adapt to external changes.
+
+## Deliverables
+
+- develop common instrumentation libraries for popular GenAI clients and frameworks
+- suggest and, when possible, contribute them to upstream libraries
+- define and evolve GenAI Semantic Conventions
+- propose and implement changes in the OTel API/SDK/contrib/etc that are necessary for GenAI efforts
+
+## Staffing / Help Wanted
+
+The initial set of approvers for each language must include:
+
+- representatives of at least 2 different observability vendors that are active members of OTel LLM SemConv WG
+- at least one existing approver or maintainer of the contrib, instrumentation or core OTel repo in the corresponding language
+
+**Project Leads:**
+
+- @drewby
+- @nirga
+
+**Sponsoring Members:**
+
+- @lmolkova
+- TBD
+
+**Engineers:**
+- Python
+  - maintainers/approvers:
+    - @drewby
+    - @karthikscale3
+    - @lzchen
+    - @nirga
+  - contributors:
+    - @alizenhom
+    - @lmolkova
+    - *looking for more contributors*
+- JS
+  - maintainers/approvers:
+    - @drewby
+    - @karthikscale3
+    - @nirga
+    - *looking for existing OTel JS approvers/maintainers for general JS & OTel feedback*
+  - contributors:
+    - @alizenhom
+    - *looking for more contributors*
+- Other languages: not in the scope (yet).
+
+## Meeting Times
+
+Alternating weekly meetings to accommodate different time zones:
+
+- Week 1: Wednesdays at 10 AM PST
```

Comment:
Would it be possible to move this to 9AM PST? It would be a bit more easier to partecipate from central europe timezones :)

Reply:
Sure, I updated the doc - will start a poll to find more times. BTW we have another meeting at 8am Thu CET, would that one work for you?

Reply:
Does the meeting time change (for rest of world) when the clocks move back an hour in November?

Reply:
I believe time stays the same in the timezone it's specified. 

https://github.com/open-telemetry/community/blob/ad757c73a375e59211ae0986f707f6e9de76d848/README.md?plain=1#L130-L131